fastlane/metadata/android/de-DE/changelogs/11.txt

```diff
@@ -1,5 +1,5 @@
-- Wenn du eine Karten-ID bearbeitest, wird die vorhandene ID zu Beginn ausgefüllt. (Pull #94 (https://github.com/brarcher/loyalty-card-locker/pull/94))
-- Begrenzen der Breite der erzeugten Barcodes, um die Speichernutzung und Fehler durch Speicherüberschreitung zu reduzieren. (Pull #103 (https://github.com/brarcher/loyalty-card-locker/pull/103))
-- Beim Bearbeiten einer Karte wird die Schaltfläche "Karte eingeben" in "Karte bearbeiten" geändert, wenn bereits eine Karten-ID existiert. (Pull #104 (https://github.com/brarcher/loyalty-card-locker/pull/104))
-- Farbschema angepasst so, dass mit dem App-Symbol kompatibel ist und das Layout beim Anzeigen einer Karte übersichtlicher gestaltet. (Pull #107 (https://github.com/brarcher/loyalty-card-locker/pull/107))
-- Hinzufügen eines Intro-Assistenten, der beim ersten Start der App gestartet wird. (Pull #108 (https://github.com/brarcher/loyalty-card-locker/pull/108))
+- Karten-ID beim Bearbeiten vorab ausfüllen. (Pull #94 [Link](https://github.com/brarcher/loyalty-card-locker/pull/94))
```

Comment:
FYI @arbadacarbaYK, fastlane changelogs are text files, it doesn't support Markdown.

I've removed the `[Link]` text on Weblate for now as that'll only confuse users.